designer/client/context/__tests__/FeatureFlagContext.jest.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { Router } from "react-router-dom";
+import { act } from "react-dom/test-utils";
+import { FeatureFlagProvider } from "../FeatureFlagContext";
+import { createMemoryHistory } from "history";
+import sinon from "sinon";
+import { FeatureToggleApi } from "../../api/toggleApi";
+import FeatureToggle from "../../FeatureToggle";
+import { useFeatures } from "../../hooks/featureToggling";
+const history = createMemoryHistory();
+history.push("");
+
+const customRender = (ui, { providerProps, ...renderOptions }) => {
+  const rendered = render(
+    <Router history={history}>
+      <FeatureFlagProvider value={providerProps}>{ui}</FeatureFlagProvider>
+    </Router>,
```

Comment:
Do you need `<Router>` for this test?

Reply:
NOOO, good point. I made a mental note to remove it, but I forgot how forgetful I am 😂